File: FunnyAlgorithms-master/FizzBuzz/FizzBuzz.go
package main

import (
	"fmt"
)

func main() {
	for i := 1;  i<=100; i++ {
		fmt.Print(i, " ->");
		if (i % 3 == 0) {
			fmt.Print(" Fizz");
		}
		
		if (i % 5 == 0) {
			fmt.Print(" Buzz");
		}
		fmt.Println("");
	}
}

File: FunnyAlgorithms-master/FizzBuzz/fizzbuzz.vb
'this is the fizzbuzz programe in visual basic
   
   Function fb(num)
        If num Mod 15 = 0 Then
            Console.WriteLine("fizzbuzz")
        ElseIf num Mod 3 = 0 Then
            Console.WriteLine("fizz")
        ElseIf num Mod 5 = 0 Then
            Console.WriteLine("buzz")
        Else
            Console.WriteLine(num)
        End If
    End Function
    Sub Main()
        Dim num As Integer
        num = Console.ReadLine()
        fb(num)
        Console.ReadKey()
    End Sub
